src/xtgeo/cube/cube1.py

```diff
@@ -919,8 +782,15 @@ def to_file(self, sfile, fformat="segy", pristine=False, engine="xtgeo"):
 
         fobj.check_folder(raiseerror=OSError)
 
+        if engine is not None:
+            warnings.warn(
+                "Providing an 'engine' value is no longer supported and will have no "
+                "effect in the future. segyio will eventually be used by default.",
+                UserWarning,
+            )
```

Comment:
A sneak attached change

Reply:
Good, but perhaps say that xtgeo built-in applied now as default (I had problems with segyio doing that a while ago, and need to re-investigate)

Reply:
Done